### DISCUSSION TITLE
In addition to the initial backbone, what other backbones can be used with patchcore?

### DISCUSSION THREAD

**Discussion by praycxy** [open-edge-platform/anomalib#1874] [Q&A] [answered]
`model:
  class_path: anomalib.models.Patchcore
  init_args:
    backbone: wide_resnet50_2
    layers:
      - layer2
      - layer3
    pre_trained: true
    coreset_sampling_ratio: 0.1
    num_neighbors: 9`

**Comment by samet-akcay** [comment, marked as answer]
You could refer to this how-to-guide on the documentation. Let us know if you have any other questions.
https://anomalib.readthedocs.io/en/latest/markdown/guides/how_to/models/feature_extractors.html

**Reply by praycxy** [reply]
thank you very much!
